sketcherMinimizer.cpp

```diff
@@ -3607,4 +3609,93 @@ int sketcherMinimizer::morganScores(const vector<sketcherMinimizerAtom*>& atoms,
     return n;
 }
 
+// interactions with m_minimizer and m_fragmentBuilder
+std::vector<sketcherMinimizerStretchInteraction*> sketcherMinimizer::getStretchInteractions()
+{
+    return m_minimizer.getStretchInteractions();
+}
+
+std::vector<sketcherMinimizerInteraction*> sketcherMinimizer::getInteractions()
+{
+    return m_minimizer.getInteractions();
+}
+
+std::set<sketcherMinimizerAtom*> sketcherMinimizer::getChetoCs(
+    const std::vector<sketcherMinimizerAtom*>& allAtoms)
+{
+    return m_minimizer.getChetoCs(allAtoms);
+}
+
+std::set<sketcherMinimizerAtom*> sketcherMinimizer::getAminoNs(
+    const std::vector<sketcherMinimizerAtom*>& allAtoms)
+{
+    return m_minimizer.getAminoNs(allAtoms);
+}
+
+std::set<sketcherMinimizerAtom*> sketcherMinimizer::getAlphaCs(
+    const std::vector<sketcherMinimizerAtom*>& allAtoms,
+    const std::set<sketcherMinimizerAtom*>& chetoCs,
+    const std::set<sketcherMinimizerAtom*>& aminoNs)
+{
+    return m_minimizer.getAlphaCs(allAtoms, chetoCs, aminoNs);
+}
+
+void sketcherMinimizer::clearInteractions()
+{
+    m_minimizer.clearInteractions();
+}
+
+void sketcherMinimizer::addInteractionsOfMolecule(
+    sketcherMinimizerMolecule* molecule, bool intrafragmentClashes)
+{
+    m_minimizer.addInteractionsOfMolecule(molecule, intrafragmentClashes);
+}
+
+void sketcherMinimizer::minimizeMolecule(sketcherMinimizerMolecule* molecule)
+{
+    m_minimizer.minimizeMolecule(molecule);
+};
+
+void sketcherMinimizer::run()
```

Comment:
maybe we can take the chance to rename this function. In my head "sketcherMinimizer" should be named something like "Coordgen", while this runs the (usually) final minimization  that relaxes the structure by allowing atoms to move around and bonds to bend/stretch

Reply:
@ZontaNicola  - Are you proposing that we rename the class `sketcherMinimizer` to `Coordgen`, basically?

Reply:
I think something like that would be a good idea!

Reply:
yes!

Reply:
and also that we rename ::run() into ::relaxStructure() or ::forceFieldMinimize() ::runFFMinimization() or something like that

Reply:
yeah, I guess `forceFieldMinimize()` is the best option.